src/mlpack/methods/ann/layer/c_relu.hpp

```diff
@@ -84,7 +94,7 @@ class CReLUType : public Layer<InputType, OutputType>
 // Convenience typedefs.
 
```

Comment:
Since the `CReLU` output size is not the same size as the input, we'll need to implement `ComputeOutputDimensions()`.  If I am not mistaken it should be just twice the size of the input.